external-crates/move/crates/move-vm-runtime/src/cache/arena.rs

```diff
@@ -19,7 +19,7 @@ pub struct Arena(Bump);
 /// An arena-allocated vector. Notably, `Drop` does not drop the elements it holds, as that is the
 /// perview of the arena it was allocated in.
 #[derive(Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
-pub struct ArenaVec<T>(std::mem::ManuallyDrop<Vec<T>>);
+pub struct ArenaVec<T>(pub(crate) std::mem::ManuallyDrop<Vec<T>>);
```

Comment:
I'd probably keep this field private to keep a bit more of an abstraction barrier here, and then add a `drain` (or `drain_all`) function to the impl for `ArenaVec` and use that.